truss-chains/truss_chains/public_types.py

```diff
@@ -473,6 +473,7 @@ class WebSocketProtocol(Protocol):
 
     async def close(self, code: int = 1000, reason: Optional[str] = None) -> None: ...
 
+    async def receive(self) -> Union[str, bytes]: ...
```

Comment:
Context for this change is here: https://basetenlabs.slack.com/archives/C06CZ3RSXRU/p1757360869605229